app-connector-client/package.json

```diff
@@ -6,13 +6,16 @@
   "scripts": {
     "start": "node server/server",
     "build": "docker build -t varkes/server .",
-    "test": "mocha"
+    "test": "nyc mocha dist/test",
+    "ts": "npm run copy && tsc && node dist/server/server",
```

Comment:
For me it is not clear when I should call what. Please assure we have simple commands like, build, test, start